apps/admin/frontend/src/app.test.tsx

```diff
@@ -86,7 +85,7 @@ beforeEach(() => {
   apiMock.expectGetMachineConfig();
   apiMock.expectGetSystemSettings();
 
-  MockDate.set(new Date('2020-11-03T22:22:00'));
+  jest.setSystemTime(new Date('2020-11-03T22:22:00'));
```

Comment:
Jest's "modern" fake timers include `Date` mocking, so this PR replaces `mockdate` with that.